Documentation/content/examples/index.md

```diff
@@ -63,32 +64,130 @@ This will allow you to see the some live code running in your browser. Just pick
 [SceneExplorerVesselWithStreamlines]: ../docs/gallery/SceneExplorerVesselWithStreamlines.jpg
 [SkyboxViewer]: ../docs/gallery/SkyboxViewer.jpg
 [VolumeViewer]: ../docs/gallery/VolumeViewer.jpg
+[QuadView]: ../docs/gallery/QuadView.jpg

```

Comment:
the image looks squeezed. Can you re-take it and respect the ratio ? (please remove the "stats" bottom-left corner layout)